extra/1-currency-conversion.js

```diff
@@ -5,7 +5,11 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(PoundAmount) {
+  let usdAmount = (PoundAmount * 1.4)
```

Comment:
Is there a reason you're using let instead of const?
if you're not going to reassign the value again, you should always declare variables with const as a general rule

Reply:
Thank you Seyoung for your guidance. I'll changed it.